docs/SOPs/HCA_Metadata_Spreadsheet_Guide_Wranglers.md

```diff
@@ -313,23 +303,23 @@ The highest level biological entity (often Donor) is not ‘derived from’ anyt
 
 One process entity is automatically created between every biomaterial, protocol, and file entity in the graph. A process entity contains metadata about the transition between entities. The process itself contains the who, when, where metadata which is linked to protocol entities with detailed metadata about the methods. One process can link to zero, one, or more protocols.
 
-![image alt text](../../images/ssheet_guidance_image_3.png)
+![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_3.png)
 
 ### Protocols
 
 Protocols need to be linked to the processes sandwiched between biomaterial and/or file entities.
 
-![image alt text](../../images/ssheet_guidance_image_4.png)
+![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_4.png)
 
 To create a link from one collection protocol to 5 processes between ‘donor organism’ and ‘specimen from organism’, we add the collection protocol ID to the specimen tab. **We always add protocol IDs to the lower entity in the hierarchy**.
 
-![image alt text](../../images/ssheet_guidance_image_5.png)
+![image alt text](https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/assets/images/spreadsheet_guidance/ssheet_guidance_image_5.png)
 
 Although this sample shows the linking of one protocol, multiple protocols can be added to a process.
 
 ### Bundles
 
-Datasets are not submitted to the data store or analyzed by the secondary analysis pipelines as a whole; instead, datasets are organized into bundles. The splitting is roughly technology dependent as files are grouped together based on requirements for each analysis pipeline run. Bundling for technologies that do not yet have a pipeline should be done consistently with other datasets in the HCA. If you are unsure you should consult with the pipeline developers and other wranglers to define the ideal bundle.
+Datasets are not submitted to the data store, instead datasets are organized into bundles. The splitting is roughly technology dependent as files are grouped together based on requirements for each analysis pipeline run. Bundling for technologies that do not yet have a pipeline should be done consistently with other datasets in the HCA. If you are unsure you should consult with the pipeline developers and other wranglers to define the ideal bundle.
```

Comment:
We no longer have analysis pipelines, maybe we should remove that part entirely?

Reply:
I removed the reference with this last commit